Resources/views/CRUD/edit_orm_many_association_script.html.twig

```diff
@@ -34,6 +34,10 @@ This code manage the many-to-[one|many] association field popup
         initialize_popup_{{ id }}();
 
         var target = jQuery(this);
+        
+        if (target.attr('href').length === 0 || target.attr('href') === '#') {
```

Comment:
> What do you mean by "top level" exactly?

[window.top](https://developer.mozilla.org/en-US/docs/Web/API/Window/top)

Url-related properties of anchors (a.href, a.origin...) are resolved against it's window's location (`a.ownerDocument.defaultView.location`).

If the page were loaded in an iframe, solution 1. would work by doing this:
```js
const isAnchor = a => {
  const loc = a.ownerDocument.defaultView.location
  return a.tagName === 'A'
    && a.origin === loc.origin
    && a.pathname === loc.pathname
    && a.search === loc.search
}
```

But again, since we load the page via ajax, a.href will always be resolved against the top-level window.

Now to be even more accurate, solution n°2 could look more like:
```js
// String -> String
const withoutHash = url => url.replace(/#(.*)$/, '')
// String -> HTMLElement -> Boolean
const isAnchorToUrl = (url, a) => {
  const href = (a.getAttribute('href') || '').trim()
  return a.tagName === 'A' && (
    !href // empty href
    || href[0] === '#' // hash only
    // same urls, hash-insensitive
    || withoutHash(url) === withoutHash(href)
    // after that, things start getting hairy...
  )
}
```
Which still doesn't handle all possible cases...
However I think the short version of n°2 can be considered good enough for the use-case.